src/Activation/ActivationSummary.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { FunctionComponent } from "react"
+import {
+  TouchableOpacity,
+  Linking,
+  ScrollView,
+  View,
+  StyleSheet,
+  Image,
+} from "react-native"
+import { useTranslation } from "react-i18next"
+
+import { usePermissionsContext } from "../PermissionsContext"
+import { useOnboardingContext } from "../OnboardingContext"
+import { useApplicationName } from "../More/useApplicationInfo"
+import { GlobalText } from "../components"
+import { Button } from "../components"
+import { useBluetoothStatus } from "../useBluetoothStatus"
+import { useHasLocationRequirements } from "../Home/useHasLocationRequirements"
+
+import { Images } from "../assets"
+import { Buttons, Colors, Spacing, Typography } from "../styles"
+
+const NotificationsPermissions: FunctionComponent = () => {
+  const { t } = useTranslation()
+  const { applicationName } = useApplicationName()
+  const { completeOnboarding } = useOnboardingContext()
+  const { isLocationOffAndNeeded } = useHasLocationRequirements()
+  const isBluetoothOn = useBluetoothStatus()
+
+  const { exposureNotifications } = usePermissionsContext()
+  const { status } = exposureNotifications
+  const { authorized, enabled } = status
+
+  const isProximityTracingOn = authorized && enabled
+  const isAppSetupComplete =
+    isProximityTracingOn && isBluetoothOn && !isLocationOffAndNeeded
+
+  const handleOnPressGoToHome = () => {
+    completeOnboarding()
+  }
+
+  const handleOnPressOpenSettings = async () => {
+    Linking.openSettings()
+    completeOnboarding()
+  }
+
+  const image = isAppSetupComplete
```

Comment:
What do you think about turning these into a configuration object and then using one ternary to return the proper object?

```javascript
const appSetupCompleteContent = {
  headerImage: Images.CheckInCircle,
  headerText: ...,
  bodyText: determineBodyText(isAppSetupComplete, isLocationOffAndNeeded)
}

const screenContent = isAppSetupComplete ? appSetupCompleteContent : appSetupIncompleteContent

<Image source={screenContent.image} />
```

Reply:
Good idea!